_docs/javascript-api.md

```diff
@@ -1561,6 +1582,35 @@ var Activity = Java.use("android.app.Activity");
 var activity = Java.cast(ptr("0x1234"), Activity);
 {% endhighlight %}
 
+**Note** Difference between Java.use() and Java.choose() 
+
+*use () :*
+Doest not give direct access to the objects at all. 
+Gives access to the object via the `this` operator.
```

Comment:
The explanation was just my understanding, which I agree does not give a clear picture at all. You explanation definitely makes much better sense. Will use that.